pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -668,6 +670,17 @@ func toKubeContainerStatus(status *runtimeapi.ContainerStatus, runtimeName strin
 		cStatusUser = toKubeContainerUser(status.User)
 	}
 
+	var cStatusStopSignal *v1.Signal
+	if utilfeature.DefaultFeatureGate.Enabled(features.ContainerStopSignals) {
+		if signal := status.GetStopSignal().String(); signal != "" {
+			cStatusStopSignal = fromCRIStopSignal(status.GetStopSignal())
+		} else {
+			cStatusStopSignal = nil
+		}
+	} else {
+		cStatusStopSignal = nil
+	}
+
```

Comment:
```suggestion
var cStatusStopSignal *v1.Signal
if utilfeature.DefaultFeatureGate.Enabled(features.ContainerStopSignals) {
    signal := status.GetStopSignal().String()
    if signal != "" {
        cStatusStopSignal = fromCRIStopSignal(status.GetStopSignal())
    }
}
```
This will make the code look simpler.